index.html

```diff
@@ -89,7 +90,7 @@ <h2 class="section-heading text-uppercase">Past Events</h2>
             <td>
               <div class="hero-box mt-3 align-self-center">
                 <a href="{{ post.url | absolute_url }}">
-                  <img src="assets/img/logos/datasaturday_temp.png" height="65" />
+                  <img src="assets/design/logo/11.jpg" height="65" />
```

Comment:
This one, @SQLDBAWithABeard

Reply:
gotcha, I was tired. looks like this

![image](https://user-images.githubusercontent.com/6729780/114349776-3eefca00-9b60-11eb-87c2-dcd502d909e7.png)